packages/front-end/components/Settings/UpgradeModal/index.tsx

```diff
@@ -257,6 +285,115 @@ export default function UpgradeModal({ close, source }: Props) {
     }
   };
 
+  const upgradeHeader = (
+    <>
+      <h3 className="mb-1">Upgrade to Pro</h3>
+      <p
+        className="mb-0"
+        style={{ color: "var(--color-text-mid", fontSize: "16px" }}
+      >
+        Get instant access to advanced experimentation, permissioning and
+        security features.
+      </p>
+    </>
+  );
+
+  function trialAndUpgradeTreatment() {
+    return (
+      <div>
+        {upgradeHeader}
+        <div className="py-4">
+          <RadioCards
+            columns="2"
+            value={trialAndUpgradePreference}
+            setValue={(v) => setTrialAndUpgradePreference(v)}
+            options={[
+              {
+                value: "trial",
+                label: "Pro Trial",
+                description: "Free for 14 days",
+              },
+              {
+                value: "upgrade",
+                label: "Pro",
+                description: "Upgrade now. No credit card required.",
+              },
+            ]}
+          />
+        </div>
+        <p style={{ color: "var(--color-text-mid" }}>
+          Continuing will{" "}
+          {trialAndUpgradePreference === "trial"
+            ? "start a 14-day Pro plan trial for your organization."
+            : "require a credit card to purchase a Pro plan for your organization."}
+          <a
+            href="https://www.growthbook.io/pricing"
+            className="text-decoration-none pl-1"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <strong className="a link-purple text-decoration-none">
+              Learn more <FaExternalLinkAlt className="pl-1" />
+            </strong>
+          </a>
+        </p>
+      </div>
+    );
+  }
```

Comment:
For both of these treatments, we might want to add a callout for Enterprise that links to https://www.growthbook.io/demo.  Want to keep the main focus on Pro, but still provide an escape hatch for those who are looking for enterprise.

> Interested in an Enterprise plan?  [Talk to Sales]